src/xenialdan/Backpack/Loader.php

```diff
@@ -219,6 +224,19 @@ public function onEnable()
 	{
 		@mkdir($this->getDataFolder() . "players");
 		$this->getServer()->getPluginManager()->registerEvents(new EventListener(), $this);
+
+		if (!PacketHooker::isRegistered()) {
+			PacketHooker::register($this);
+		}
+		if (!InvMenuHandler::isRegistered()) {
+			InvMenuHandler::register($this);
+		}
+		if (InvMenuHandler::getMenuType(self::MENU_TYPE_BACKPACK_CHEST) === null) {
+			$copy = InvMenuHandler::getMenuType(InvMenu::TYPE_CHEST);
```

Comment:
why the copy?

Reply:
BackpackMenuMetadata holds similar data to the MenuMetadata that InvMenu::TYPE_CHEST is mapped to, i guess $parent makes more sense